src/backend/routers/paras.ts

```diff
@@ -0,0 +1,59 @@
+import { z } from "zod";
+import { procedure } from "../trpc";
+
+export const paraProcedures = {
+  getParaById: procedure
+    .input(z.object({ user_id: z.string().uuid() }))
+    .query(async (req) => {
+      console.log("req", req);
+      const { user_id } = req.input;
+
+      const result = await req.ctx.db
+        .selectFrom("user")
+        .where("user_id", "=", user_id)
+        .selectAll()
+        .executeTakeFirst();
+
+      return result;
+    }),
+
+  getAllParas: procedure.query(async (req) => {
+    const result = await req.ctx.db.selectFrom("user").selectAll().execute();
+
+    return result;
+  }),
+
+  createPara: procedure
```

Comment:
This can be shared with creating a CM flow as well.

Right now, that is is adapter.ts -> createdUser, which can likely be refactored into users.ts (being introduced in PR 55)

Reply:
This function can be removed. An entry in the user table is created when someone logs in.

Reply:
I see what you mean - just for clarity's sake, would that imply that we have an "invite para" button in place of a create para button for when a case manager is looking to add a para to their roster? And/or would we just have an "add para" button that searches through the existing list of users? @amantri

Reply:
Ah - I see what you mean. Yeah, if its a new para, we will want to create a row for them in the user table when the CM adds them. If they already are in the system, we can reuse their user_id. I think you probably need a call to getUserByEmail in auth/adapter.ts here.

Reply:
After thinking it through a bit more I think I will make a separate PR to change the functionality of creating a para so that we can consolidate the paras.ts with the users.ts first (which I think Tessa is going to handle in a separate PR) @amantri